CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## Unreleased
+## v1.0.0 (August 23, 2023)
```

Comment:
A final without https://github.com/cosmos/iavl/pull/805 ?

Reply:
true missed that pr, ill wait for it